weave-js/src/components/PagePanelComponents/Home/Browse3/pages/MonitorsPage/MonitorsPage.tsx

```diff
@@ -0,0 +1,208 @@
+              renderCell: params => (
+                <CellValue value={`${params.value * 100}%`} />
+              ),
+            },
+            {
+              field: 'ops',
+              headerName: 'Ops',
+              flex: 1,
+              valueGetter: (_, row) => row.obj.val['op_names'],
+              renderCell: params => {
+                const opRefs: string[] = params.value;
+                return opRefs.length > 0 ? (
+                  <div className="flex items-center gap-2">
+                    {opRefs[0] !== ALL_TRACES_OR_CALLS_REF_KEY ? (
+                      <SmallRef
+                        objRef={parseRef(opRefs[0]) as WeaveObjectRef}
+                      />
+                    ) : null}
+                    {opRefs.length > 1 ? (
+                      <span>{`+${opRefs.length - 1}`}</span>
+                    ) : null}
+                  </div>
+                ) : null;
+              },
+            },
+            {
+              field: 'scorers',
+              headerName: 'Scorers',
+              flex: 1,
+              valueGetter: (_, row) => row.obj.val['scorers'],
+              renderCell: params => {
+                const scorerRefs: string[] = params.value;
+                return scorerRefs.length > 0 ? (
+                  <div className="flex items-center gap-2">
+                    <CellValue value={scorerRefs[0]} />
+                    {scorerRefs.length > 1 ? (
+                      <span>{`+${scorerRefs.length - 1}`}</span>
+                    ) : null}
+                  </div>
+                ) : null;
+              },
+            },
+            {
+              field: 'active',
+              headerName: 'Active',
+              valueGetter: (_, row) => {
+                return row.obj.val['active'];
+              },
+              renderCell: params => {
+                return <CellValue value={params.value} />;
+              },
+            },
+            {
+              field: 'callCount',
+              headerName: 'Calls',
+              valueGetter: (_, row) => {
+                return row.id;
+              },
+              renderCell: params => (
+                <CallCountCell
+                  monitorRef={params.value}
+                  entity={entity}
+                  project={project}
+                />
+              ),
+            },
+          ]}
+        />
+      </Box>
+      <CreateMonitorDrawer
+        entity={entity}
+        project={project}
+        open={isCreateDrawerOpen}
+        onClose={() => setIsCreateDrawerOpen(false)}
+        monitor={selectedMonitor}
+      />
+    </Tailwind>
+  );
+};
+
+const CallCountCell = ({
+  monitorRef,
+  entity,
+  project,
+}: {
+  monitorRef: string;
+  entity: string;
+  project: string;
+}) => {
+  const query: Query = useMemo(() => {
+    return {
+      $expr: {
+        $contains: {
+          input: {$getField: 'feedback.[*].trigger_ref'},
```

Comment:
reminder - another place for that magic string - probably a search and replace would be good.